app/views/forms/index.haml

```diff
@@ -14,4 +14,6 @@
             form_draft_path(draft), method: :delete
         - else
           = link_to 'Edit', new_form_draft_path(form_id: form.id)
-.actions= button_to 'New form', nil, disabled: true
+      %td= link_to 'Delete', form_path(form.id), method: :delete,
```

Comment:
Sorry, that was only half a comment. What you *should* be able to do is just pass the object itself, removing the `.id`.